.github/workflows/test.yml

```diff
@@ -28,6 +28,13 @@ jobs:
       run: diff <(echo -n) <(gofmt -d .)
       if: matrix.os == 'ubuntu-latest'
 
+    - name: Install dockexec
+      run: go install mvdan.cc/dockexec@latest
+      if: matrix.os == 'ubuntu-latest'
+    - name: Test interpreter confirm
+      run: CGO_ENABLED=0 go test -run TestRunnerRunConfirm -exec 'dockexec bash:5.1' ./interp/.
```

Comment:
I would instead have one step that runs two commands, calling it something like `Run confirm tests with Bash 5.1`.

Also, you can just use `./interp` as the final argument.